templates/status/404.tmpl

```diff
@@ -1,8 +1,8 @@
 {{template "base/head" .}}
-<div role="main" aria-label="{{.Title}}" class="page-content ui container center gt-w-screen {{if .IsRepo}}repository{{end}}">
+<div role="main" aria-label="{{.Title}}" class="page-content ui center gt-w-screen {{if .IsRepo}}repository{{end}}">
```

Comment:
Does `ui` or `container` make sense here? The `gt-w-screen` seems a no-op here too.

Maybe it should be: `class="page-content {{if .IsRepo}}repository{{end}}"`